app/src/main/java/com/hyorim/sopt_assigmnet_1/SignUpActivity.kt

```diff
@@ -0,0 +1,56 @@
+package com.hyorim.sopt_assigmnet_1
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.hyorim.sopt_assigmnet_1.databinding.ActivitySignUpBinding
+
+class SignUpActivity : AppCompatActivity() {
+    var Tag = "SignUpActivity :"
+    private lateinit var binding: ActivitySignUpBinding
+    private lateinit var name   : String
+    private lateinit var id     : String
+    private lateinit var pw     : String
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        name    = binding.nameEditText.text.toString()
+        id      = binding.IDEditText.text.toString()
+        pw      = binding.PWEditText.text.toString()
+
+        /**Sign Up Button*/
+        binding.signUpBtn.setOnClickListener {
+            if (isInputComplete()){
+                val intent = Intent(this, SignInActivity::class.java)
+                intent.putExtra("id", id)
+                intent.putExtra("pw", pw)
```

Comment:
이렇게도 표현 가능할 것 같습니다~!!
```suggestion
intent.putExtra("id", id)
          .putExtra("pw", pw)
```